src/app/ffbe/model/skill.model.spec.ts

```diff
@@ -685,3 +687,27 @@ export const ABILITY_SKILLS_SHORTDESCRIPTIONS_TEST_DATA =
         "Aprende Ración triple durante un turno al inicio del combate o tras ser revivido, y aumenta poco a poco el daño de determinadas habilidades tras 2 turnos"
     ]
   }`;
+
+describe('Skill', () => {
+  it('should not consider effect as damaging for fixed damages to caster', () => {
```

Comment:
Is it the right place to add a fixture?
This file is included in many other tests.

Reply:
C'est le bon endroit pour mettre le test unitaire.
Ce n'est peut être pas le meilleur endroit pour mettre les donnés de test. Ceci dit, elles ne posent pas de problème (du moins technique, pour la lisibilité on pourrait envisager de les déplacer dans un fichier `skill.testdata.spec.ts`.